Client.xcodeproj/project.pbxproj

```diff
@@ -6310,10 +6310,11 @@
 			buildSettings = {
 				CODE_SIGN_ENTITLEMENTS = "$(SRCROOT)/Extensions/Entitlements/FennecEnterprise.entitlements";
 				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
-				DEVELOPMENT_TEAM = "";
+				DEVELOPMENT_TEAM = 9G8J6YA743;
```

Comment:
@st3fan was this accidental commit?

Reply:
@garvankeeley looks like it - but this commit is months old .. is it causing trouble?

Reply:
i am in one of those hellish states where xcode suddenly won't run on my device "iPhone has denied the launch request", for like an hour now. I am just grasping at straws here.

Reply:
to clarify: this is not the problem, i just noticed it when looking for other problems